File: Week1/problem_set/mario/mario_more.c
#include <stdio.h>
#include <cs50.h>

int main(void)

{
    //Promt user for height
    int x;

    do
    {
        x = get_int("Height: ");
    }
    while (x < 1 || x > 8);

    //Print pyramid on the left
    for (int y = 1; y <= x; y++)
    {
        for (int i = x; i > y; i--)
        {
            printf(" ");
        }
        for (int j = 0; j < y; j++)
        {
            printf("#");
        }
        //Make a gap between pyramids
        {
            printf("  ");
        }
        //Print pyramid on the right
        for (int k = 0; k < y; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Week6/problem_set/mario/more/mario_more.py
from cs50 import get_int


def main():
    while True:
        pyramid_height = get_int("Pyramid height: ")
        if pyramid_height >= 1 and pyramid_height <= 8:
            break

    number_of_spaces = pyramid_height - 1
    number_of_hashes = 1
    for i in range(pyramid_height):
        for j in range(number_of_spaces - i):
            print(" ", end="")
        for n in range(number_of_hashes):
            print("#", end="")
        print("  ", end="")
        for m in range(number_of_hashes):
            print("#", end="")
        print()
        number_of_hashes += 1


if __name__ == "__main__":
    main()